Vagrantfile

```diff
@@ -9,7 +9,8 @@ Vagrant.configure("2") do |config|
     override.vm.hostname = "quadlet.example.com"
 
     override.vm.provider "libvirt" do |libvirt, provider|
-      libvirt.memory = "4096"
+      libvirt.memory = 10240
+      libvirt.cpus = 4
```

Comment:
I've split this out to https://github.com/theforeman/foreman-quadlet/pull/48, as it's "technically" unrelated